androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/pages/ExplainerPageTests.kt

```diff
@@ -0,0 +1,93 @@
+package com.mbta.tid.mbta_app.android.pages
+
+import androidx.compose.ui.test.hasText
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.performClick
+import com.mbta.tid.mbta_app.android.stopDetails.TripRouteAccents
+import com.mbta.tid.mbta_app.model.ObjectCollectionBuilder
+import com.mbta.tid.mbta_app.model.RouteType
+import com.mbta.tid.mbta_app.model.stopDetailsPage.ExplainerType
+import kotlin.test.assertTrue
+import org.junit.Rule
+import org.junit.Test
+
+class ExplainerPageTests {
+    @get:Rule val composeTestRule = createComposeRule()
+
+    @Test
+    fun testCloses() {
+        var closed = false
+        composeTestRule.setContent {
+            ExplainerPage(
+                ExplainerType.FinishingAnotherTrip,
+                TripRouteAccents.default,
+                goBack = { closed = true },
+            )
+        }
+
+        composeTestRule.waitForIdle()
+
+        composeTestRule.onNodeWithContentDescription("Close").performClick()
+        assertTrue(closed)
+    }
+
+    @Test
+    fun testFinishingAnother() {
+        val objects = ObjectCollectionBuilder()
+        composeTestRule.setContent {
+            ExplainerPage(
+                ExplainerType.FinishingAnotherTrip,
+                TripRouteAccents(objects.route { type = RouteType.LIGHT_RAIL }),
+                goBack = {},
+            )
+        }
+
+        composeTestRule.waitForIdle()
+        composeTestRule.onNode(hasText("Finishing another trip"))
```

Comment:
These should probably be `onNodeWithText().assertDisplayed()` - I don't think `onNode()` with no further calls makes any assertion at all.